yt_dlp/extractor/afreecatv.py

```diff
@@ -76,59 +76,6 @@ class AfreecaTVIE(InfoExtractor):
             },
         }],
         'skip': 'Video is gone',
-    }, {
-        'url': 'http://vod.afreecatv.com/PLAYER/STATION/18650793',
-        'info_dict': {
-            'id': '18650793',
-            'ext': 'mp4',
-            'title': '오늘은 다르다! 쏘님의 우월한 위아래~ 댄스리액션!',
-            'thumbnail': r're:^https?://.*\.jpg$',
-            'uploader': '윈아디',
-            'uploader_id': 'badkids',
-            'duration': 107,
-        },
-        'params': {
-            'skip_download': True,
-        },
-    }, {
-        'url': 'http://vod.afreecatv.com/PLAYER/STATION/10481652',
```

Comment:
Replace them with tests having similar URL

Reply:
They are all gone and all the URL formats are consolidated to single format `https://vod.afreecatv.com/player/[0-9]*`, so new test case covers all current urls.